styles/Footer.module.scss

```
@@ -0,0 +1,58 @@
+.footer {
+    padding: 2rem 0;
+    position: relative;
+    line-height: 1.5;
+    text-align: center;
+    width: 100%;
+    overflow: hidden;
+
+    @include media-query(medium) {
+        align-self: flex-end;
+        display: flex;
+        align-items: center;
+        width: auto;
+    }
+
+    .copyright {
+        margin-right: 0.25rem;
+
+        @include media-query(medium) {
+            font-weight: $weight--bold;
+        }
+    }
+
+    &__text {
+        display: flex;
+        flex-direction: column;
+        position: relative;
+        z-index: 1;
+        @include font-size(xxs);
```

```suggestion
    &__text {
        @include font-size(xxs);
        display: flex;
        flex-direction: column;
        position: relative;
        z-index: 1;
```

**Nitpick**: Nice, this is all looking good. Ben Morse taught me recently that `@include` rules which aren't media queries should go to the top of the class definition, I like this idea of laying out styles. We could add another ticket to Codebase for setting up a [Stylelint config](https://stylelint.io/) for this project which should catch things like this automatically.